browser/brave_vpn/win/wireguard_utils_win.cc

```diff
@@ -79,14 +83,21 @@ bool EnableBraveVpnWireguardServiceImpl(const std::string& config) {
   }
   DWORD error_code = 0;
   if (FAILED(service->EnableVpn(
-          base::UTF8ToWide(base::Base64Encode(config)).c_str(), &error_code))) {
+          ::SysAllocString(base::UTF8ToWide(server_public_key).c_str()),
```

Comment:
why are we calling `SysAllocString` here when we apparently didn't have to before?

Reply:
it's particularly concerning since I don't see any calls to `SysFreeString` so it seems like this will leak?

Reply:
Fixed in https://github.com/brave/brave-core/pull/24261/commits/c48a27fc395cedc277ccced2fab406a8ca5c13e8